_posts/2025-05-20-firefox-security-privacy-newsletter-2025-q1.md

```diff
@@ -0,0 +1,72 @@
+
+**CSS Fuzzing:** We are also attributing a major piece of our platform goals to [Interop 2025](https://wpt.fyi/interop-2025): Enhancements in CSS syntax generation have shown immediate results and will continue to support the work throughout the year.
+
+**XSLT Fuzzing**: Early in Q1, we have also received an experimental XSLT fuzzer from Ivan Fratric of Google Project Zero that is now running continuously in our lab. Thank you, Ivan\!
+
+**NSS Fuzzing:** Our fuzzing focus on NSS (Network Security Services) has also shown a significant uptick in NSS fuzzing coverage across both oss-fuzz as well as our internal tools. We also added a TSan target for NSS, ensuring stricter thread-safety going forward.
+
+**Site-scout**: Site-scout is a tool that visits web pages with instrumented builds (like Address Sanitizer). We are continuing to enhance site-scout, such that it interacts with websites and ideally tease out more bugs.
+
+## Community Engagement
+
+**Bug Bounty:** The [Firefox client bug bounty program](https://www.mozilla.org/en-US/security/client-bug-bounty/) rewards contributors for finding security bugs in Firefox. Our award categories range between $500 to $20,000 depending on bug severity and report quality. With 31 awarded bounties this quarter, it is one of our main sources of security bug reports. The majority of the reported bugs are of *moderate* severity UI spoofs. We intend to change the reward structure for these kinds of bugs going forward \- please find details on our [Firefox bug bounty pages](https://www.mozilla.org/en-US/security/client-bug-bounty/)**.** We are also opening the opportunity for **guest blog posts** on attackanddefense.dev. [Please reach out if you want to write about a previous finding of yours\!](https://attackanddefense.dev/about/#guest-blog-posts)
+
+**Mozilla Research Summit:** The [Mozilla Research Summit](https://surf.mozilla.org/events/2025/sandiego/) took place on March 1st, 2025 in San Diego California. We had 56 participants from 3 continents and 25+ different research institutions. The day included some high-caliber talks from Mozillians as well as security and privacy academics and sparked new research collaborations between Mozilla and the academic community.
+
+**Co-organized Open-Source Crypto Summit**: We co-organized the [Open Source Cryptography Workshop (OSCW)](https://opensourcecryptowork.shop/) which brings together practitioners interested in all sorts of open source crypto projects ranging from production and maintenance challenges to best practices for designing clean-sheet crypto systems with modern primitives.
```

Comment:
Are there slides somewhere? I don't see them on the site. Anybody we could poke?